scripts/run_benchmark.py

```diff
@@ -0,0 +1,31 @@
+#!/bin/env/python
+"""Script to run the benchmark on a vector database."""
+
+from pathlib import Path
+from typing import Annotated
+
+import typer
+
+from inatinqperf.benchmark.benchmark import Benchmarker
+
+app = typer.Typer()
+
+
+@app.command("run")
+def main(
+    config_file: Annotated[
+        Path, typer.Argument(help="The configuration file to use for running the benchmark.")
+    ],
+    base_path: Annotated[
+        Path, typer.Option(help="The base path relative to which various artifacts are saved.")
+    ] = Path(__file__).parent.parent,
+) -> None:
```

Comment:
Consider `typer.Option(ensure_exists=True, file_okay=False, help="The base path relative to which various artifacts are saved.")`.